front-end/src/screens/OnChain/Referenda/query.ts

```diff
@@ -1,23 +1,15 @@
 import gql from 'graphql-tag';
-// import { post_type } from '../../../global/post_types';
-// import { post_topic } from '../../../global/post_topics';
 
-// const type = post_type.ON_CHAIN;
-// const topic = post_topic.DEMOCRACY;
+// post_type.ON_CHAIN === 2
 
 export const QUERY_LATEST_REFERENDA = gql`
-  query LatestDemocracyReferendaPosts($postType: Int! = 2, $postTopic: Int! = 1, $limit: Int! = 5) {
+  query LatestReferendaPosts($postType: Int! = 2, $limit: Int! = 5) {
```

Comment:
shouldn't we keep postType 2 in constant const ON_CHAIN_TYPE = 2

Reply:
no because a motion post has the type `council`.

Reply:
yes i am just saying rather than keeping magic numbers here we are defining them in const and using everywhere. Code is more readable that way

Reply:
I don't think you can do this in a GQL, I tried as far as I remember. I'll check it out again.